cmd/replicate-add.go

```diff
@@ -57,6 +59,10 @@ var replicateAddFlags = []cli.Flag{
 		Name:  "remote-bucket",
 		Usage: "remote bucket, should be a unique value for the configuration",
 	},
+	cli.StringFlag{
+		Name:  "replicate",
+		Usage: "replicate delete markers, versioned deletes or both (delete-marker, delete)",
```

Comment:
This usage doc comment is confusing.

Reply:
`Comma-separated list to enable replication of delete markers and/or deletion of versioned objects.`

Reply:
updated